### ISSUE TITLE
Some propositions to improve the GUI and text of the PYTHONPATH manager

### ISSUE THREAD

**Issue by jnsebgosselin** [spyder-ide/spyder#10728]
### Issue Report Checklist

1. I would remove the window context help button because it doesn't do anything.
2. I would keep the 'Ok' button enabled even when no change was made. I mean, there is no drawback if the user press 'Ok' without having done any change. I think disabling the button send a message that something is wrong or missing (I was confuse, a little,  by this when testing previously). We can still make the check internally before applying the paths though.
3. I would change the way we ask users if they want or not to 'clear the contents of PYTHONPATH before synchronizing' by using a checkbox. I think this would be less confusing. The state of the checkbox could be saved in the configs.

![image](https://user-images.githubusercontent.com/10170372/68970727-f3dc5680-07b5-11ea-8f0c-af0afb73eec6.png)

**Comment by goanpeca** [comment]
I like 1 and 3 but I think 2 serves a clear purpose. If things have not chabged there should not be the possibility to “apply” anything

**Comment by jnsebgosselin** [comment]
> 
> 
> I like 1 and 3 but I think 2 serves a clear purpose. If things have not chabged there should not be the possibility to “apply” anything

Yes exactly, but it's not an `Apply` button, it is an `Ok` button.  I would be ok with this reasoning if we used an `Appy` and `Close` buttons instead of an `Ok` and `Cancel` buttons. Besides, what is the real drawback if the user clicks on `Ok` while no change were made?

While testing the synchronize button, I started up with all paths checked. Then I unchecked and checked some paths while testing the synchronize functionality and after a couple of tests I checked again all the paths and I was like `Why is this preventing me from clicking on `Ok`, this is exactly what I want`. I had to guess that it was probably because I started up with all paths checked in the first place.

Anway, it's not a big deal, it's only my 2-cents here.

**Comment by jnsebgosselin** [comment]
Another argument in favor of keeping the `Ok` button enabled would be for Ux consistency with the `Preferences dialog`.
